host/tee_u_wrapper.c

```diff
@@ -31,14 +34,8 @@ static oe_result_t _call_tee_enclave_function(
         output_bytes_written);
 }
 
-/* Ignore missing edge-routine prototypes. */
-#if defined(__GNUC__)
-#pragma GCC diagnostic ignored "-Wmissing-prototypes"
-#endif
-
```

Comment:
I would expect removing this to cause a compiler error

Reply:
The edger8r has been fixed to add the `static` keyword for local functions. edger8r already generates prototypes in header files for non local functions.

Reply:
ok. I would generally prefer this (and the modification to static) to be a separate change since they're pretty much completely separate from the core change.

Reply:
At least in a separate commit